server/basicServices/basicServices/GenericRepresentation.js

```diff
@@ -145,6 +149,14 @@ exports.getResponseValueList = function (operationName) {
                                 responseInstanceValue = value[onfAttributes.RESPONSE_PROFILE.STATIC_VALUE];
                             }
                             responseInstancedataTypeOfValue = typeof responseInstanceValue;
+
+                            if(fieldName['static-field-name'] == fieldNameDataUpdatePeriod){
```

Comment:
can we declare a separate function for this functionality if possible to make code readable .

Reply:
done